### PR TITLE
DOC+ENH: fixing minor docstring issues

### PR DESCRIPTION
**Pull request by dohmatob**
addresses issues #731, #729, #730, ...

### CODE REVIEW

File: nilearn/decoding/space_net_solvers.py [hunk 1]
```diff
@@ -140,12 +141,11 @@ def _graph_net_adjoint_data_function(X, w, adjoint_mask, grad_weight):
...
 
     grad_weight: float
-        l1_ratio * alpha
+        l1_ratio * alpha.
 
     Returns
     -------
-    float
-
+    ndarray
```
Comment (AlexandreAbraham) [inline nilearn/decoding/space_net_solvers.py]: I think that the comment @bthirion made is precisely to have more that the type here. We want to know exactly the semantics of the returned value.

**Comment by bthirion** [comment]
LGTM

File: nilearn/decoding/searchlight.py [hunk 2]
```diff
@@ -186,10 +188,11 @@ def _group_iter_search_light(list_rows, estimator, X, y,
     return par_scores
 
 
-##############################################################################
-### Class for search_light ###################################################
-##############################################################################
+#
```
Comment (lesteve) [inline nilearn/decoding/searchlight.py]: Looks like you have the same version of autopep8 as Alex that removes lines of hashes like that. Try pip install -U and/or revert these changes.

File: nilearn/decoding/space_net.py [hunk 3]
```diff
@@ -205,12 +205,14 @@ def _space_net_alpha_grid(X, y, eps=1e-3, n_alphas=10, l1_ratio=1.,
 
 
 class _EarlyStoppingCallback(object):
+
```
Comment (lesteve) [inline nilearn/decoding/space_net.py]: My version of autopep8 doesn't add a new line here

File: nilearn/decoding/space_net.py [hunk 4]
```diff
@@ -457,6 +459,7 @@ def path_scores(solver, X, y, mask, alphas, l1_ratios, train, test,
 
 
 class BaseSpaceNet(LinearModel, RegressorMixin, CacheMixin):
+
```
Comment (lesteve) [inline nilearn/decoding/space_net.py]: Same thing, I don't think the newline is needed here.

File: nilearn/decoding/space_net.py [hunk 5]
```diff
@@ -457,6 +459,7 @@ def path_scores(solver, X, y, mask, alphas, l1_ratios, train, test,
...
     """
     Regression and classification learners with sparsity and spatial priors
```
Comment (lesteve) [inline nilearn/decoding/space_net.py]: You may as well fix the docstring while you are at it: Regression should be on the same line as the first `"""`

File: nilearn/decoding/space_net.py [hunk 6]
```diff
@@ -919,6 +934,7 @@ def predict(self, X):
 
 
 class SpaceNetClassifier(BaseSpaceNet):
+
```
Comment (lesteve) [inline nilearn/decoding/space_net.py]: No newline needed

File: nilearn/decoding/space_net.py [hunk 7]
```diff
@@ -1097,6 +1114,7 @@ def _binarize_y(self, y):
 
 
 class SpaceNetRegressor(BaseSpaceNet):
+
```
Comment (lesteve) [inline nilearn/decoding/space_net.py]: idem

File: nilearn/decoding/tests/test_same_api.py [hunk 8]
```diff
@@ -97,10 +97,10 @@ def test_lipschitz_constant_loss_logreg():
 
 
 def test_graph_net_and_tvl1_same_for_pure_l1(max_iter=100, decimal=2):
-    ###############################################################
+    #
```
Comment (lesteve) [inline nilearn/decoding/tests/test_same_api.py]: bring back the line of hashes

Comment (dohmatob) [inline nilearn/decoding/tests/test_same_api.py]: Oops! Indeed I had autopep8 0.9.1 from debian. Just installed 1.2. Thx for
spotting these weirdos!

On Thu, Jul 30, 2015 at 8:44 AM, Loïc Estève <notifications@github.com>
wrote:

> In nilearn/decoding/tests/test_same_api.py
> <https://github.com/nilearn/nilearn/pull/739#discussion_r35842783>:
>
> > @@ -97,10 +97,10 @@ def test_lipschitz_constant_loss_logreg():
> >
> >
> >  def test_graph_net_and_tvl1_same_for_pure_l1(max_iter=100, decimal=2):
> > -    ###############################################################
> > +    #
>
> bring back the line of hashes
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nilearn/nilearn/pull/739/files#r35842783>.
>



-- 
DED

**Comment by lesteve** [comment]
Hmmm sorry for that, looks like this one needs a rebase now that the autopep8 examples got merged.

**Comment by lesteve** [comment]
LGTM, merging thanks!

**Comment by lesteve** [comment]
@dohmatob do you mind closing the issue referenced here. You probably know right off the bat whether they can be closed now.
